spec/services/catalog/update_order_item_spec.rb

```diff
@@ -0,0 +1,75 @@
+describe Catalog::UpdateOrderItem do
+  describe "#process" do
+    let(:request) do
+      { :headers => { 'x-rh-identity' => encoded_user_hash }, :original_url => 'whatever' }
+    end
+
+    around do |example|
+      ManageIQ::API::Common::Request.with_request(request) { example.call }
+    end
+
+    let(:topic) { ManageIQ::Messaging::ReceivedMessage.new(nil, nil, payload, nil) }
```

Comment:
Ahh - here is the failure. Add `client` as the final argument here and the specs will go green again.

Reply:
Hmm, I had this locally but it was failing because it's only expecting 4 arguments locally. Guess I need to update something?